ngraph_onnx/onnx_importer/ops_bridge/opset_01.py

```diff
@@ -462,34 +464,25 @@ def Greater(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> N
 def And(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
     """Perform the `and` logical operation elementwise on two input tensors."""
     left, right = broadcast_for_binary_operation(onnx_node, ng_inputs)
-    left = ng.convert(ng.not_equal(left, 0), int)
-    right = ng.convert(ng.not_equal(right, 0), int)
-    return ng.convert(left * right, bool)
+    return ng.logical_and(left, right)
 
 
 def Or(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
     """Perform the `or` logical operation elementwise on two input tensors."""
     left, right = broadcast_for_binary_operation(onnx_node, ng_inputs)
-    left = ng.convert(ng.not_equal(left, 0), int)
-    right = ng.convert(ng.not_equal(right, 0), int)
-    return (left + right) > 0
+    return ng.logical_or(left, right)
```

Comment:
Can we use `ng.convert` to `bool` when the incoming type is not a `bool`?
We could support more cases then. We can also raise a warning.